src/main/java/Main.java

```diff
@@ -13,9 +13,11 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
 public class Main {
-    private static Injector injector = Injector.getInstance("cinema");
+    private static final AnnotationConfigApplicationContext CONTEXT
```

Comment:
по суті контекст - це не константа, тому можна і так
```suggestion
    private static final AnnotationConfigApplicationContext context
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names